### PR TITLE
p1-modal initial commit

### PR DESCRIPTION
**Pull request by anshulraheja**
Modal Component

### CODE REVIEW

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    background:var(--MODAL-BACKGROUND-COLOR);
+    box-shadow: 0px 0px 10px 0px var(--MODAL-SHADOW-COLOR);
+    padding: 1rem; 
+    border-radius: 0.5rem;
+    width: 50%;
+    min-width: 200px;
+    opacity: 1;
+}
+.modal-header{
+    display: flex;
```
Comment (pankaj-prog) [inline components/modal/modal.css:16]: leave one empty line before starting new block, makes the code readable

**Review by pankaj-prog — COMMENTED** [review]
Give the spacing between elements in the documentation. Spacing is very important for enhancing the user experience.

File: components/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                        <li><a href="../slider/slider.html">Slider</a></li>
+                        <li><a href="../text-utilities/textUtilities.html">Text Utilities</a></li>
+                        <li><a href="../toast/toast.html">Toast</a></li>
+                    </ul>
+                </div>
+            </div>
+            <!-- documentation -->
+            <div class="documentation-container">
+                <h1>Modal</h1>          
+                <div class="modal">
```
Comment (SinchanaMS) [inline components/modal/modal.html:63]: Suggestion: A bit of margin between these two elements in your documentation would look good. 
The rest looks good to merge.

File: components/modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    <i class="fas fa-bars"></i>
+                </div>
+                <div>
+                    <i class="fas fa-strikethrough"></i>
+                    <a href="../../index.html">Scale UI</a>
+                </div>
+            </div>
+            <ul class="right-nav">
+                <li><a href="https://github.com/anshulraheja/" target="_blank"><i class="fab fa-github-square"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/anshul-raheja/" target="_blank"><i class="fab fa-linkedin"></i></a></li>
```
Comment (deekshithmd) [inline components/modal/modal.html:26]: It will look good if you increase size of icon for small extent

**Review by j24m — APPROVED** [review]
Looks good to me:)

File: components/modal/modal.html (hunk 2 above)
Comment (m26unkwn) [reply components/modal/modal.html:63]: +1

**Review by m26unkwn — APPROVED** [review]

**Pull request merged**